### PR TITLE
fix tests issue #178

### PR DESCRIPTION
**Pull request by axce1**
new PR for #178

### CODE REVIEW

File: elizabeth/providers/personal.py [hunk 1]
```diff
@@ -34,6 +34,8 @@ def __init__(self, *args, **kwargs):
         self._store = {
             'age': 0
         }
+        print('# # # # # # # # # # # # # # # # # # ')
```
Comment (sobolevn) [inline elizabeth/providers/personal.py]: Are these lines required?

**Comment by axce1** [comment]
@sobolevn fix it

File: elizabeth/providers/personal.py (hunk 1 above)
Comment (axce1) [inline elizabeth/providers/personal.py]: oops!!! fix

**Comment by sobolevn** [comment]
Thanks! Let's wait for @lk-geimfari to merge it!

**Comment by lk-geimfari** [comment]
@sobolevn You can merge any PR by yourself, if everything is in okay. You are collaborator.

**Pull request merged**